docs/get-started/getting-started-kubernetes.md

```diff
@@ -0,0 +1,256 @@
+---
+layout: page
+title: Kubernetes
+nav_order: 1
+parent: Getting-Started
+---
+
+# Getting Started with RAPIDS and Kubernetes
+
+This guide will run through how to set up the RAPIDS Accelerator for Apache Spark in Kubernetes Cluster.
+At the end of this guide, the reader will be able to run a sample Apache Spark application that runs
+on NVIDIA GPUs in Kubernetes Cluster.
+
+Kubernetes requires a Docker image to run Spark.  Generally everything needed is in the Docker
+image - Spark, the RAPIDS Accelerator for Spark jars, and the discovery script.  See this
+[Dockerfile.cuda](Dockerfile.cuda) example.
+
+
+## Prerequisites
+    * Kubernetes Cluster is up and running with NVIDIA GPU support
+    * Docker is installed on a client machine
+    * A Docker repository which is accessible by Kubernetes Cluster
+
+These instructions do not cover how to setup a Kubernetes cluster.
+
+Please refer to [Install Kubernetes](https://docs.nvidia.com/datacenter/cloud-native/kubernetes/install-k8s.html) on 
+how to install a Kubernetes Cluster with NVIDIA GPU support.
+
+## Docker Image Preparation
+
+On a client machine which has access to the Kubernetes Cluster:
+
+1. Download Spark from https://spark.apache.org/downloads.html
+   Supported versions of Spark are listed on the [download](../download.md) page.  Please note that only
+   scala version 2.12 is currently supported by the accelerator. 
+
+   Note that you can download these into a local directory and untar the Spark `.tar.gz` as a directory named `spark`.
+
+2. Download [RAPIDS Accelerator for Spark jars](getting-started-on-prem.md#download-the-rapids-jars), and the
+  [GPU discovery script](getting-started-on-prem.md#install-the-gpu-discovery-script).
+  
+   Put the 2 jars -- `rapids-4-spark_<version>.jar`, `cudf-<version>.jar`  and `getGpusResources.sh` in the same directory as `spark`.
+
+3. Download the sample [Dockerfile.cuda](Dockerfile.cuda) in the same directory as `spark`.
+
+   Basically the sample Dockerfile.cuda will copy the `spark` directory's several sub-directories into `/opt/spark/`, 
+   and copy above 2 jars and `getGpusResources.sh` into `/opt/sparkRapidsPlugin` inside the Docker image.
+   
+   So please look into the Dockerfile.cuda to make sure the file names are correct, and modify if needed.
+   
+   Currently the directory in the local machine should look as below:
+   ```shell 
+   $ ls
+   Dockerfile.cuda   cudf-<version>.jar   getGpusResources.sh   rapids-4-spark_<version>.jar   spark
+   ```
+
+4. Build the Docker image with proper repository name and tag and push to repository
+   ```shell 
+   export IMAGE_NAME=xxx/yyy:tag
+   docker build . -f Dockerfile.cuda -t $IMAGE_NAME
+   docker push $IMAGE_NAME
+   ```
```

Comment:
Do we really require the user build a docker image to run this in Kubernetes? I would expect an automated build in docker hub and this doc to distill down to `kubectl apply -f https://github.com/NVIDIA/rapids/examples/k8s/deply.yaml` or, is complex, `helm install ...` and an example workload.

Reply:
@hyperbolic2346  this could one goal. Currently this guide is to give users a way to customize the docker image if they want.